examples/common/scripts/k3s-down.sh

```diff
@@ -16,11 +16,7 @@
 
 # This is an example script that stops the k3s server started by k3s-up.sh.
 
-set -e
```

Comment:
should we keep `set -e`, i don't see it in `env.sh`

Reply:
Good catch! Didn't intend to do that. Will put it back.